### PR TITLE
fix extra /

### PR DESCRIPTION
**Pull request by KhrulkovV**
## Description
The argument `logdir` in [CheckpointCallback](https://github.com/catalyst-team/catalyst/blob/b27d8def595b8974ad255792d969a043233c09df/catalyst/dl/callbacks/checkpoint.py#L173) is slightly incorrect which results in extra `/`. E.g., 
`logs/resnet101_28_5e-05_0.001_480_True/checkpoints//train.0.pth 0.4945`

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the [Code of Conduct](https://github.com/catalyst-team/catalyst/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I have read the [Contributing](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md) guide.
- [ ] I have checked the code-style using `make check-style`.
- [ ] I have written the docstring in Google format for all the methods and classes that I used.
- [ ] I have checked the docs using `make check-docs`.

### CODE REVIEW

**Comment by TezRomacH** [comment]
Hi, could you please wrap the logdirs to `pathlib.Path` without manual removing the slash?

**Comment by Scitator** [comment]
@KhrulkovV @TezRomacH Please uncheck WIP label after the discussion and final implementation :)

**Comment by KhrulkovV** [comment]
> Hi, could you please wrap the logdirs to `pathlib.Path` without manual removing the slash?

Fixed!

File: catalyst/utils/checkpoint.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-from typing import Dict  # isort:skip
-from collections import OrderedDict
+from typing import Dict, Union
```
Comment (TezRomacH) [inline catalyst/utils/checkpoint.py]: please add `# isort:skip`

Comment (KhrulkovV) [reply catalyst/utils/checkpoint.py]: Done.

**Review by TezRomacH — APPROVED** [review]

**Pull request merged**